codec-http/src/main/java/io/netty/handler/codec/http/HttpContentCompressor.java

```diff
@@ -84,24 +82,20 @@ public HttpContentCompressor(int compressionLevel, int windowBits, int memLevel)
      * Creates a new handler with the specified compression level, window size,
      * and memory level..
      *
-     * @param compressionLevel
-     *        {@code 1} yields the fastest compression and {@code 9} yields the
-     *        best compression.  {@code 0} means no compression.  The default
-     *        compression level is {@code 6}.
-     * @param windowBits
-     *        The base two logarithm of the size of the history buffer.  The
-     *        value should be in the range {@code 9} to {@code 15} inclusive.
-     *        Larger values result in better compression at the expense of
-     *        memory usage.  The default value is {@code 15}.
-     * @param memLevel
-     *        How much memory should be allocated for the internal compression
-     *        state.  {@code 1} uses minimum memory and {@code 9} uses maximum
-     *        memory.  Larger values result in better and faster compression
-     *        at the expense of memory usage.  The default value is {@code 8}
-     * @param contentSizeThreshold
-     *        The response body is compressed when the size of the response
-     *        body exceeds the threshold. The value should be a non negative
-     *        number. {@code 0} will enable compression for all responses.
+     * @param compressionLevel     {@code 1} yields the fastest compression and {@code 9} yields the
+     *                             best compression.  {@code 0} means no compression.  The default
+     *                             compression level is {@code 6}.
+     * @param windowBits           The base two logarithm of the size of the history buffer.  The
+     *                             value should be in the range {@code 9} to {@code 15} inclusive.
+     *                             Larger values result in better compression at the expense of
+     *                             memory usage.  The default value is {@code 15}.
+     * @param memLevel             How much memory should be allocated for the internal compression
+     *                             state.  {@code 1} uses minimum memory and {@code 9} uses maximum
+     *                             memory.  Larger values result in better and faster compression
+     *                             at the expense of memory usage.  The default value is {@code 8}
+     * @param contentSizeThreshold The response body is compressed when the size of the response
+     *                             body exceeds the threshold. The value should be a non negative
+     *                             number. {@code 0} will enable compression for all responses.
```

Comment:
it would be easier if you not do any reformatting that is not related to your change to review

Reply:
Apologies on behalf of my IDE. :-|